pkg/datapath/linux/probes/probes.go

```diff
@@ -196,10 +207,12 @@ func (p *ProbeManager) SystemKernelHz() (int, error) {
 // warnings when optional parameters are not enabled.
 func (p *ProbeManager) SystemConfigProbes() error {
 	config := p.features.SystemConfig
-	// Required
-	if !config.ConfigBpf.Enabled() {
-		return fmt.Errorf("CONFIG_BPF kernel parameter is required")
```

Comment:
Since your current version for `KernelConfigAvailable()` checks the presence of the config file and no longer `CONFIG_BPF` is enabled, we should probably restore that check. Subsequent checks on e.g. `CONFIG_BPF_SYSCALL` will fail if `CONFIG_BPF` is disabled, but it would help users to know that this is `CONFIG_BPF` we should look at in such a case.

Reply:
Yeah this makes sense.